plugins/woocommerce/src/Blocks/BlockTypes/ProductCollection/ProductCollectionController.php

```diff
@@ -0,0 +1,428 @@
+<?php
+declare(strict_types=1);
+
+namespace Automattic\WooCommerce\Blocks\BlockTypes\ProductCollection;
+
+use Automattic\WooCommerce\Blocks\BlockTypes\AbstractBlock;
+use InvalidArgumentException;
```

Comment:
Looks like this is not used anywhere. I think we can remove it.

Reply:
Fixed in commit [06acc33](https://github.com/woocommerce/woocommerce/pull/52429/commits/06acc3331538ee176d2ccf95846df85335840427)!